backend/.env

```diff
@@ -1,5 +1,6 @@
 REST_API_PORT=5000
 API_DOCS_ENDPOINT_URL=/rest-api-docs
+GOOGLE_CLIENT_ID=1052788207529-fjnskiqrsm09i9kbujp3gmtasnp21su7.apps.googleusercontent.com
```

Comment:
To nie moze byc w repo, bo teraz kazdy moze sobie tego uzyc. Musi byc placeholder, ale to tylko w zespole przekazane. A normalnie potem na serwerze.